apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableVersion.cs

```diff
@@ -23,74 +23,93 @@ namespace Google.Cloud.Bigtable.V2
     /// <remarks>
     /// <para>
     /// Note: when using ReadModifyWriteRow, modified columns automatically use a server version, which
```

Comment:
I think it's probably worth explaining why both millis and micros are referred to here at all. I'm certainly confused about it, having only paid half attention to what's going on.

Is micros the raw underlying format, but the server only ever sets values in millis??

Reply:
That's correct. Ok, I'll update all these comments.

Reply:
Actually, I think I'll make another note preceding this one to explain the whole thing.